### PR TITLE
Mark line(s) to share highlighted part(s) of the paste

### PR DESCRIPTION
**Pull request by 9mido**
@Tigerpanzer02 is the author of this PR

Attempts to solve https://github.com/PrivateBin/PrivateBin/issues/743

Test here:

https://paste.plugily.xyz

I wrote my opinions in the issue of how I think this should be improved because I did not quite like it completely when testing. It works, but it is hard to use from a UX standpoint / has some bugs. But is a good starting point.

I am helping @Tigerpanzer02 make a PR request so that others can help improve this code to make it more user friendly. I will not be able to make any coding changes. So feel free to close this and make an improved PR since I cannot edit the code and make changes.

### CODE REVIEW

**Comment by rugk** [comment]
Hi @9mido and @Tigerpanzer02,
thanks for your first contribution to this project! :tada: :+1: 🏅
I hope you'll like this project and enjoy hacking on it… :smiley:

First of all, thanks for bringing it into a PR form. Of course, it would be better if @Tigerpanzer02 created that PR for the reasons you stated.

BTW: Next time, try to avoid doing a pull request from the master branch, because you can run into problems when you have a "non-clean" master that does not follow this repo here (i.e. "upstream"). [See this article for details](https://help.github.com/en/articles/proposing-changes-to-your-work-with-pull-requests). Anyway, this is only a tip for the next time. :smiley:

And you can (automatically) [let issues close](https://help.github.com/articles/closing-issues-via-commit-messages/) when a PR is merged by adding some "magic" text to your PR _body_. (Manually linked it now.)

But so far off-topic stuff…

File: tpl/bootstrap.php [hunk 1]
```diff
@@ -59,41 +59,29 @@
...
+		<link rel="apple-touch-icon" href="img/apple-touch-icon.png?<?php echo rawurlencode($VERSION); ?>" sizes="180x180" />
+		<link rel="icon" type="image/png" href="img/favicon-32x32.png?<?php echo rawurlencode($VERSION); ?>" sizes="32x32" />
+		<link rel="icon" type="image/png" href="img/favicon-16x16.png?<?php echo rawurlencode($VERSION); ?>" sizes="16x16" />
 		<link rel="manifest" href="manifest.json?<?php echo rawurlencode($VERSION); ?>" />
-		<link rel="mask-icon" href="img/safari-pinned-tab.svg" color="#ffcc00" />
+		<link rel="mask-icon" href="img/safari-pinned-tab.svg?<?php echo rawurlencode($VERSION); ?>" color="#ffcc00" />
 		<link rel="shortcut icon" href="img/favicon.ico">
 		<meta name="msapplication-config" content="browserconfig.xml">
 		<meta name="theme-color" content="#ffe57e" />
-		<!-- Twitter/social media cards -->
```
Comment (rugk) [inline tpl/bootstrap.php:85]: Your code seems to revert a lot, we already had in master, i.e. it seems to reuse old code from an old (non-development) version of PrivateBin.
As such, it's really hard to review and not possible to merge in that state, unfortunately.

**Review by rugk — CHANGES_REQUESTED** [review]
Thanks for the test website.

Testing it, I still see some usability issues.
grafik
grafik
grafik

It's good for a quick mockup, but the text buttons there are a little too much IMHO.
Multi-line selecting should preferably work with drag- and dropping.
And single-line selection just can be a click, no need for a label.
Also, clearing the selecting, dunno… may be an extra button.
The term "scrolling" might also not be what we'd imagine.

As discussed in the issue too.

**Comment by 9mido** [comment]
@Tigerpanzer02 I made the PR directly from the github website I did not use any command line tools. What you need to do is fork privatebin, navigate to your fork, make a branch, make your code changes in the appropriate files by clicking the pencil edit icon on the coding files, then navigate back to the main privatebin github page, click on pull requests, and you should be able to submit a pull request from the branch you made. You need to be the one to make the PR since I am not a PHP programmer. 

When you do this, I think we should close this PR and work off of your PR instead. Or if someone from the privatebin team could make a draft PR based on what is seen here, that would work too.

**Comment by Tigerpanzer02** [comment]
Can be closed... See #789

**Pull request closed without merging**